test/Serilog.Tests/Serilog.Tests.csproj

```diff
@@ -32,4 +35,7 @@
   <PropertyGroup Condition=" '$(TargetFramework)' == 'net7.0' ">
     <DefineConstants>$(DefineConstants);FEATURE_DEFAULT_INTERFACE;FEATURE_SPAN;FEATURE_DATE_AND_TIME_ONLY;FEATURE_ITUPLE;FEATURE_ASYNCDISPOSABLE</DefineConstants>
```

Comment:
perhaps make a DefineConstantsGeNet60 and refer to that in the 3 supported ones?
that makes it easy to see a) if there is variance b) the specifics of that variance where one of the later ones needs a new flag due to stressing some new API specific to a later RTM

Reply:
Prefer avoiding this if possible, just in favor of a simpler setup.

Reply:
Makes sense ;)